_javascripts/page-loader.js

```diff
@@ -26,7 +26,7 @@ function versionSelector(list) {
     baseUrl = "https://docs.openshift.com/enterprise/";
   }
 
-  if (dk === "openshift-enterprise" && currentVersion.charAt(0) !== newVersion.charAt(0)){
+  if (dk === "openshift-enterprise" || dk === "openshift-origin" && currentVersion.charAt(0) !== newVersion.charAt(0)){
```

Comment:
```suggestion
  if ((dk === "openshift-enterprise" || dk === "openshift-origin") && currentVersion.charAt(0) !== newVersion.charAt(0)){
```

Reply:
To be 100% sure, it's better to place the parenthesis around the OR statement.

Reply:
Hi @aireilly ,
Is there anything I should add or modify on the pull request?

Reply:
Need to see a build output that verifies the change please.

Reply:
Sure, understandable. Can I trigger that or that's automatic?

Reply:
It's a manual and laborious process that I wouldn't wish on anyone :sweat: 
https://file.emea.redhat.com/aireilly/fix-okd-version-switcher/welcome/index.html

Change lgtm!